IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,229 @@
+    - The proposed Powpeg will be eliminated.
+    - The election will be allowed once again.
+
+- SVP period didn't end        
+    - If the funding transaction signed hash is registered in the FUND_TX_HASH_SIGNED storage entry and the Proof transaction is not created yet, the Bridge will:
+        - Create it as described [here](#proof-transaction)
+        - Add it to a new storage set, key `spendTxWaitingForSignatures`
+        - Set it in a new Storage entry:
+            ```
+            KEY: "SPEND_TX_HASH"
+            VALUE: hash of SVP proof tx
+            ```    
+
+##### Proof transaction
+
+The Proof transaction is a Bitcoin transaction where a UTXO from the proposed Powpeg is sent to the current Powpeg.
+This transaction will be as simple and small as possible to avoid spending much satoshis as fees.
+
+Proof transaction structure:
+```
+{
+    inputs: [
+        {
+            txid: FUND_TX_HASH_SIGNED,
+            output_index: FUNDING_TX_OUTPUT_INDEX
+        }
+    ],
+    outputs: [
+        {
+            script: CURRENT_POWPEG_OUTPUT_SCRIPT,
+            value: TOTAL_UTXO - FEES
+        }
+    ]
+}
+```
+
+#### Proof transaction confirmation
+
+The SVP proof transaction should go through a new "peg-out signing process", since it has to be signed for the proposed pegnatories which belong neither the active nor the retiring powpeg, but the proposed one.
+
+#### Proof transaction signing
+
+The proof transaction should be signed as any other peg-out, but by the proposed powpeg. For this to happen, a new Bridge method `ADD_SPEND_TX_SIGNATURE` should be created to perform the exact same actions as the `ADD_SIGNATURE` but to accept signatures from public keys that don't belong to an active/retiring Powpeg (let's recall the proposed Powpeg is not active at this point).
+The `addSpendTxSignature` method should check that the public key belongs to the proposed Powpeg and that the hash of the transaction intended to be signed exists in the `SPEND_TX_HASH` storage entry.
+
+Once fully signed, the Bridge should remove the entry from the `spendTxWaitingForSignatures` and emit the release_btc event as usual.
+
+### Proof verification
+
+With the SVP proof transaction broadcasted, the Bridge now needs to validate it and proceed with the election commitment as expected.
+
+The proof transaction should be registered through the registerBtcTransaction method. As before, it should verify:
+- SVP period is ongoing
+- Transaction hash without signatures is registered in the `SPEND_TX_HASH` storage key.
+
+If the hash matches the one registered in the storage, then the SVP process is considered completed and we can proceed with the committment of the proposed -and now validated- Powpeg.
+
+### Validation phase duration
+
+The duration of this phase must account for:
+- Spending a UTXO from the current Powpeg (Bridge confirmations + pegnatories signing)
+- Confirming the UTXO on Bitcoin (Bitcoin confirmations for the UTXO to be accepted)
```

Comment:
i think we are not checking this, are we?

Reply:
what do you mean? If there's a transaction to a federation (a pegin), a powpeg node will find it and if it has enough bitcoin confirmations it will send it to the Bridge. So, I guess we will need to just look if that utxo got registered back to the Bridge.